src/ui/server/constants/declarations/versions/latest.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * LATEST_VERSION_NUMBER
+ * Latest declaration version number.
+ * This should be manually updated when the declarations for EXIP are updated. For example:
+ * - Version number 1: MVP - no support for applications over 500k.
+ * - Version number 2: "No PDF" - Support for applications over 500k.
+ * @returns {String} Latest declaration version number
+ */
+const LATEST_VERSION_NUMBER = '2';
+
+export default LATEST_VERSION_NUMBER;
```

Comment:
Same as above, I don't think we should have this and should be based on individual declarations.

Otherwise, imagine in the future if e.g 1x single declaration is updated, just one word update. If we need to confirm what has changed in the latest version and the last version, it would be fiddly. Whereas if it's an individual declaration version, it's just one thing to check